templates/new_request_page.hbs

```diff
@@ -12,7 +12,7 @@
   </div>
 
   <h1>
-    {{t 'submit_a_request'}}
+    <div>Submit a Request or Provide Feedback</div>
```

Comment:
The submit button color is different from the live theme. NOTE: This is the new request page

Reply:
@mudassir-hafeez Can you add both the screenshots here and get a go ahead from @pdpinch ?

Reply:
Yes, here it is:

I've found some branding inconsistencies as shown in the screenshots below. These inconsistencies exist on other pages as well. So fixes for them are included in this same PR.

<img width="1250" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/3f52d2fa-6922-4aa8-a7b0-64ccc3652c01">

<img width="1250" alt="Screenshot 2024-06-03 at 11 05 09 PM" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/ceaa49c6-49d9-4ccb-bfed-1d9c115a0061">

@pdpinch, we need your approval to proceed with or revert these changes.